src/components/CodeOverview/index.spec.tsx

```diff
@@ -240,6 +246,49 @@ describe(__filename, () => {
     expect(link).toHaveProp('title', `Jump to line ${line}`);
   });
 
+  it('scrolls a linter message into view when clicking its link', () => {
+    const fakeDOMNode = { scrollIntoView: jest.fn() };
+    const _document = {
+      ...document,
+      querySelector: jest.fn().mockReturnValue(fakeDOMNode),
+    };
+
+    const root = renderWithFixedHeight({
+      _document,
+      content: generateFileLines({ count: 3 }).join('\n'),
+    });
+
+    const line = 3;
+    root
+      .find(Link)
+      .at(line - 1)
+      .simulate('click');
+
+    expect(_document.querySelector).toHaveBeenCalledWith(
+      getCodeLineAnchor(line),
+    );
+    expect(fakeDOMNode.scrollIntoView).toHaveBeenCalled();
+  });
+
+  it('does not scroll a linter message if its anchor does not exist', () => {
```

Comment:
I get what you're doing here, but I find it a bit odd that the test explicitly says "does not scroll a linter message" but then doesn't actually assert anything about that. What it does is create a situation in which an anchor does not exist, and verifies that it doesn't throw an exception, but it doesn't specifically verify that nothing was scrolled. I understand that we cannot really test that as we're not returning a DOMNode, but rather `null`, but I wonder if we should name the test differently.

Reply:
I updated the spec so it's more precise